### DISCUSSION TITLE
Submod ideas for Alpha Frontier

### DISCUSSION THREAD

**Discussion by carriontrooper** [carriontrooper/Alpha-Frontier#27] [Ideas]
Just curious, what would you want in a mod of a mod? More factions? Units, buildings, resources? Maybe a new mechanism?

**Comment by DumbDumb404** [comment]
Just keep up the good work

**Comment by Dardjan** [comment]
It would be cool to see Star Destroyers like in Star Wars. In the later stages of the game, they would be very useful. In addition, they are well combined with other units of this mod.

**Reply by Dardjan** [reply]
Было бы хорошо увидеть больше морских юнитов

**Comment by hachchch** [comment]
stockpiled Oxygen, Atomosphere, Water annd Thermal!
Correct these to Terraforming win

**Reply by carriontrooper** [reply]
Iirc you can add victories with non-ruleset mods, so yee
Especially when said resources isn't in base ruleset
